.travis.yml

```diff
@@ -5,8 +5,8 @@ matrix:
   include:
     - sudo: false
       script:
-         - ./mach test-tidy --self-test
-         - ./mach test-tidy --no-progress
+         - ./mach test-tidy --no-progress --self-test
```

Comment:
I'm not sure if this is a good idea.
I saw that `buildbot_steps.yml` has `--no-progress` with `--self-test`, and I'm assuming that they're supposed to be the same? Perhaps they just fell out-of-sync, with regard to parameters?